src/ffi_util.rs

```diff
@@ -195,6 +195,25 @@ impl<'a> CStrLike for &'a CString {
     }
 }
 
+pub struct CSlice {
+    pub data: *const c_char,
```

Comment:
The members can not be public, because they could be replaced with invalid pointers / lengths in safe code (creating bad pointer is safe, dereferencing is not), while our unsafe code relies on their validity.

Reply:
Yes. I've fixed that. (What a mistake.)